src/preparacion_cuentas_comafi.py

```diff
@@ -0,0 +1,87 @@
+import os
+import shutil
+import pandas as pd
+import numpy as np
+import datetime
+import time
+from .constants.constants import (
+    PROVINCES,
+    UTIL_COLS_COMAFI,
+)
+
+'''
+Estimados, esta función está en este momento sin implementar, pero
+va a reemplazar a la función con el mismo nombre en el archivo subida_cartera.py
+Antes de reemplazarla, quiero probar al menos una vez el software así como está,
+y para ello neces
+'''
+
+def preparacion_cuentas_comafi():
+    nombre_cartera = input('\nIngrese el nombre de la cartera que desea:  ')
+
+    print('Iniciando preparacion')
+    # lectura planilla modelo
+    df_os = pd.read_csv('modelos/modelo_cuentas.csv', encoding='ANSI', sep=';')
+    df = pd.read_excel('emerix.xlsx', dtype=str)
+
+    df = df[list(UTIL_COLS_COMAFI.keys())]
+    df = df.rename(columns=UTIL_COLS_COMAFI)
+
+    # reemplazo de valores nulos
+    df.loc[df['provincia'].isna(), 'provincia'] = '0'
+    # reemplazo de 'ñ' en nombres
+    replace_invalid_chars(df)
+    print('\n')
+    print('\nComenzando escritura de archivos..\n\n')
+
+    fill_data(df, df_os)
```

Comment:
I don't want to to change this, but., I only leave for document and remember in the future. we could return `df_os` riectly from here, and also create it inside this method. We could name the method `build_df_os( )`the only change needed is move the line [24](https://github.com/alquimistas-org/subida_cartera/pull/9/files#diff-e190420606fa8cc07fa4a12db3e90babfb8bd77c818cb057320e694de41761f1R24) inside `fille_data`

Reply:
OK, on it.